docs/topic_guides/evaluation/edges.md

```diff
@@ -1,3 +1,50 @@
 # Edge Evaluation
 
-This page is under construction - check back soon!
\ No newline at end of file
+Once you have a trained model, you use it to generate edges (links) between entities (nodes). These edges will have a Match Weight and corresponding Probability.
+
+There are several strategies for checking whether the links in created in your pipeline perform as you want/expect.
+
+## Consider the Edge Metrics
+
+Edge Metrics measure for how links perform at an overall level.
+
+First, consider how you would like your model to perform. What is important for your use case? Do you want to ensure that you capture all possible matches? Or do you want to minimise the number of incorrectly predicted matches? Perhaps a combination of both?
+
+For a summary of all the edge metrics available in Splink, check out the [Edge Metrics guide](./edge_metrics.md).
+
+!!! note
+
+    To produce Edge Metrics you will require a "ground truth" to compare your linkage results against (which can be achieved by [Clerical Labelling](./labelling.md))
+
+
+## Spot Checking pairs of records
```

Comment:
It might be useful to get Sam's views on this section as I believe he has the most experience here.

Reply:
Yes agreed - I have also requested his review